### PR TITLE
Form validation feature

### PR DESCRIPTION
**Pull request by gergitokusse**
### In this feature the following file was updated

- Index.html
- style.css
- jscript.js

### CODE REVIEW

File: jscript.js [hunk 1]
```diff
@@ -313,4 +313,84 @@ clickbtn.forEach((button) => {
...
+    emailError.innerHTML = 'It is not valide email, all letter should be in lower case !!!.';
+    emailError.className = 'error';
+  } else {
+    emailError.innerHTML = '';
+    emailError.className = 'success';
+  }
+  // validate message
+  if (messageval.validity.valueMissing) {
+    messageerror.innerHTML = 'Message filed can"t be empty!!!.';
+    messageerror.className = 'error';
```
Comment (osamaashraf6) [inline jscript.js]: - [ ] You did a great job 😃 at adding the functionality of the validation of the contact if in lowercase, but I see when I write valid information for the contact and try 💯 to submit it is not submitting it, so please try to fix that to let the users communicate with you 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| My Form | Your Form| 
| --- | --- |
|  Recording 2023-04-19 at 12 28 05 (1)  |  Recording 2023-04-19 at 12 26 33  |

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi @gergitokusse 😊, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
SoCloseAlmostGIF (2)

## Highlights🎉👌

- [x] Not linter errors present
- [x] Good descriptive pull request
- [x] Nice design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: jscript.js (hunk 1 above)
Comment (gergitokusse) [reply jscript.js]: ## Okay Thank you I will correct it.

**Review by Zuhaib042 — APPROVED** [review]
Hi @gergitokusse 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**